cli/exchange/user.go

```diff
@@ -66,18 +69,26 @@ func UserCreate(org, userPwCreds, user, pw, email string, isAdmin bool, isHubAdm
 	// get message printer
 	msgPrinter := i18n.GetMessagePrinter()
 
-	if email == "" {
-		if strings.Contains(user, "@") {
-			email = user
-		} else {
-			cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("the email must be specified via -e if the username is not an email address."))
-		}
+	var emailIsOptional bool
+	var err error
+	if emailIsOptional, err = checkExchangeVersionForOptionalUserEmail(org, userPwCreds); err != nil {
+		cliutils.Fatal(cliutils.CLI_GENERAL_ERROR, msgPrinter.Sprintf("failed to check exchange version, error: %v", err))
+	}
+
+	if email == "" && strings.Contains(user, "@") {
+		email = user
+	} else if email == "" && !emailIsOptional {
+		cliutils.Fatal(cliutils.CLI_INPUT_ERROR, msgPrinter.Sprintf("the email must be specified via -e if the username is not an email address."))
 	}
 
 	if isHubAdmin && org != "root" {
 		cliutils.Fatal(cliutils.CLI_GENERAL_ERROR, msgPrinter.Sprintf("Only exchange users in the root org can be hubadmins."))
 	}
 
+	if !isHubAdmin && org == "root" {
+		cliutils.Fatal(cliutils.CLI_GENERAL_ERROR, msgPrinter.Sprintf("Non-hubadmin user cannot be created in the root org"))
```

Comment:
Let's reword this to be positive: "Users in the root org must be hubadmins. Omit the -H option."